pkg/artifactory/resource_artifactory_local_repository_test.go

```diff
@@ -98,7 +98,7 @@ func TestAccLocalDebianRepository(t *testing.T) {
 			alias = "foo-alias{{ .kp_id }}"
 			private_key = <<EOF
 		-----BEGIN PGP PRIVATE KEY BLOCK-----
-		
+
```

Comment:
Did these tests run after the change? These keys are super sensitive to spaces and new lines

Reply:
It does IIRC. I will double check.

Reply:
I'm gonna revert the extra newlines. Look wrong even if they work.